tutorials/spanruler_restaurant_reviews/README.md

```diff
@@ -0,0 +1,113 @@
+<!-- SPACY PROJECT: AUTO-GENERATED DOCS START (do not remove) -->
+
+# 🪐 spaCy Project: Using SpanRuler for rule-based Named Entity Recognition
+
+This example project demonstrates how you can use the
+[SpanRuler](https://spacy.io/api/spanruler) component for rule-based named
+entity recognition (NER). In spaCy v3 and below, this functionality can be
+achieved via the [EntityRuler](https://spacy.io/api/entityruler). However, 
+we will start **deprecating** the `entity_ruler` component in favor of
+`span_ruler` in v4.
+
+Here, we will be using the **MIT Restaurant dataset** (Liu, et al, 2013) to
+determine entities such as *Rating*, *Location*, *Restaurant_Name*,
+*Price*, *Dish*, *Amenity*,  and *Cuisine* from restaurant reviews.
+Below are a few examples from the training data:
+
+![](figures/example_00.png)
+![](figures/example_01.png)
+![](figures/example_02.png)
+
+First, we will train an NER-only model and treat it as our baseline. Then, we will
+attach the `SpanRuler` component **before the `ner` component** of the existing
+pipeline. This setup gives us two pipelines we can compare upon.
```

Comment:
This is really nice setup and it's one of those very practical considerations that I'm sure will resonate with a lot of people. When we finalize the project, it would be nice to post something about it!

Reply:
Hi Sofie! Saw the other comment and this one is incorrect. It should be `after`. Thanks for catching this! I think I'll rerun the pipeline again and see which one performs better (adding the SpanRuler before vs. after)? Or would we prefer showing both? 

Although it seems that adding the SpanRuler after resonates with what we want to showcase in this project: SpanRuler as a helper for NER.